bin/run_nexmark.sh

```diff
@@ -21,4 +21,4 @@ VERSION=$(mvn -q \
   -Dexec.executable=echo -Dexec.args='${project.version}' \
   --non-recursive exec:exec)
 
-java -Dlog4j.configuration=file://`pwd`/log4j.properties -cp client/target/nemo-client-$VERSION-shaded.jar:`yarn classpath`:examples/nexmark/target/nexmark-$VERSION-shaded.jar org.apache.nemo.client.JobLauncher "$@"
+java -Dlog4j.configuration=file://`pwd`/log4j.properties -cp examples/nexmark/target/nexmark-${VERSION}-shaded.jar:`yarn classpath` org.apache.nemo.client.JobLauncher "$@"
```

Comment:
Could you please revert this change? 
`run_beam.sh` includes client.jar and we don't need nemo-example jar file. You should include the client jar file, instead of beam jar.  Also, nemo-360 is not related to this change. Please create another PR for separate issue.

Reply:
Done.